nomad/job_endpoint.go

```diff
@@ -2429,5 +2429,11 @@ func (j *Job) TagVersion(args *structs.JobApplyTagRequest, reply *structs.JobTag
 	}
 
 	reply.Index = index
+	if args.Tag != nil {
+		reply.Name = args.Tag.Name
+		reply.Description = args.Tag.Description
+		reply.TaggedTime = args.Tag.TaggedTime
+	}
```

Comment:
Side effect: wanted to use the POST/PUT response here and realized I was sending back instantiated-but-not-populated data.

Reply:
non-blocking: this reflects back to the user just what they passed in, but so long as we can depend on the raftApply erroring if it failed to enact their intentions, then this seems fine.